AmericanMedicalAssociation/ama-style-guide-2
# [EWL-4614] SG2 | Update Page Content Tab(s) Organism(s)

## Ticket(s)

**Jira Ticket**
- [EWL-4614: SG2 | Create Page Content Tab(s) Organism(s)](https://issues.ama-assn.org/browse/EWL-4614)

## Description
After review from UX, the page tab navigation should have a thin line underneath.

This update alters styles to achieve UX's desired look.

## To Test

- Pull `bugfix/EWL-4614-page-content-tab-organism` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Navigate to [Organism > Page Content Tabs](http://localhost:3000/?p=page-content-tabs-organisms) or [Page > Event](http://localhost:3000/?p=event-pages).
- Confirm page tab navigation looks as shown in updated Zeplin mock up.

## Visual Regressions

Updated reference images for the event page backstop tests have been included in this PR.


## Relevant Screenshots/GIFs
### Updated Event page
![image](https://user-images.githubusercontent.com/4438120/40504474-fc8840fe-5f56-11e8-8193-a2c918c6ae03.png)


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
